src/pymor/operators/constructions.py

```diff
@@ -84,44 +84,56 @@ def evaluate_coefficients(self, mu):
 
     def apply(self, U, mu=None):
         coeffs = self.evaluate_coefficients(mu)
-        R = self.operators[0].apply(U, mu=mu)
-        R.scal(coeffs[0])
+        if coeffs[0]:
+            R = self.operators[0].apply(U, mu=mu)
+            R.scal(coeffs[0])
+        else:
+            R = self.range.zeros()
```

Comment:
Is this case covered by the tests? It looks to me that it will not work when `len(U) > 1`.

Reply:
That's true. 

`self.range.zeros(len(U))`

should fix it, right?

Reply:
Yeah, I guess. Maybe we can first see if the test fails, then also add a new test.

Reply:
Ok, good idea.

Reply:
Ok, it looks like a new test would be good.

Reply:
Yes. The new test that I pushed will fail